src/background.ts

```diff
@@ -85,32 +121,43 @@ async function init (): Promise<void> {
       })
   }
 
-  await browser.notifications.create({
+  chrome.notifications.create({
```

Comment:
Will that work for Firefox? (same comment for other browser-->chrome subs in the file)

Reply:
I tried it in Firefox and it worked